talk/oms/sdk/base/win/msdkvideoencoder.cc

```diff
@@ -385,7 +455,7 @@ int MSDKVideoEncoder::Encode(
   }
 
   MSDK_ZERO_MEMORY(bs);
-  mfxU32 bsDataSize = param.mfx.FrameInfo.Width*param.mfx.FrameInfo.Height * 4;
```

Comment:
width*height*4 is not correct?

Reply:
It's OK but we don't need to allocate that much bs buffer.